app/budget/services/actions/budgetActions.js

```diff
@@ -39,6 +39,39 @@ class BudgetActions {
 						}
 					}
 
+					var sectionGroupCosts = results.sectionGroupCosts;
+					var terms = ["05", "07", "10", "01", "03"];
```

Comment:
What is this term array of 5 terms for? Why doesn't it include all 9 possible terms?

Assuming there isn't something very specialized going on, it should be called from a service to avoid bugs.